eclass/verify-sig.eclass

```diff
@@ -141,6 +141,9 @@ verify-sig_verify_detached() {
 	einfo "Verifying ${filename} ..."
 	case ${VERIFY_SIG_METHOD} in
 		openpgp)
+			# gpg can't handle very long TMPDIR
+			# https://bugs.gentoo.org/854492
+			local -x TMPDIR=/tmp
```

Comment:
does it definitely clean up after itself?

Reply:
Yep, gemato does.